public/app/features/alerting/unified/components/rules/RuleState.tsx

```diff
@@ -69,7 +69,7 @@ export const RuleState = ({ rule, isDeleting, isCreating, isPaused }: Props) =>
     return (
       <Stack gap={1}>
         <AlertStateTag state={promRule.state} isPaused={isPaused} />
-        {forTime}
+        {!isPaused && forTime}
```

Comment:
If an alert is paused, we now show a state tag of `Paused`, and we don't want to imply that it's been paused for the length of time that it might have been firing for